docs/design/2022-03-10-backup-with-batch-create-table.md

```diff
@@ -0,0 +1,117 @@
+# TiDB Design Documents
+
+- Author(s): [fengou1](http://github.com/your-github-id)
+- Discussion PR: https://github.com/pingcap/tidb/pull/27036
+- Tracking Issue: https://github.com/pingcap/tidb/issues/28763
+
+## Table of Contents
+
+* [Introduction](#introduction)
+* [Motivation or Background](#motivation-or-background)
+* [Detailed Design](#detailed-design)
+* [Test Design](#test-design)
+ * [Functional Tests](#functional-tests)
+ * [Compatibility Tests](#compatibility-tests)
+ * [Benchmark Tests](#benchmark-tests)
```

Comment:
1. Some extra spaces here.
2. Lack the section of functional tests. If there is no, remove it and clarify. Or you need to cover it

Reply:
Fix it.